static/js/translator/page.tsx

```diff
@@ -0,0 +1,240 @@
+/**
```

Comment:
nit: please keep each file to one component

Reply:
Done